cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/plugin/PluginInstantiator.java

```diff
@@ -51,6 +51,7 @@
 import io.cdap.cdap.internal.lang.Fields;
 import io.cdap.cdap.internal.lang.Reflections;
 import org.apache.twill.filesystem.Location;
+import org.elasticsearch.common.Strings;
```

Comment:
use guava over this, or just use plain java.

Reply:
oops, fixed.